docs/mobile/repo-structure.md

```diff
@@ -246,12 +220,9 @@ assets:
         title: Films
 ```
 
-* `type`, specified as `dataset`.
-* `key`, the alias for the asset.
-* `url`, the path to the asset from the root of the course repository.
-* `tabs`, with one entry for every possible tab-type, currently either `file` or
-  `table`. Every tab-type requires the sub-field `title`, which will determine
-  the name of the tab when the asset is rendered as the respective type. In the
-  example above, the asset `films_small.csv` will have the title `films.csv`
-  when rendered as a CSV file, and the title `Films` when rendered as a
-  formatted table.
+| field  | description                                                                                                                                                                                                                                                                                                                                                                                         |
+| :----  | :----------                                                                                                                                                                                                                                                                                                                                                                                         |
+| `type` | Currently, only `dataset` is supported.                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
`file` is also supported if it's a file that cannot be interpreted as a dataset

Reply:
Ah. The whole schema for tabs in the manifest is a bit confusing - we should simplify it in the next version.

Reply:
If I can give a dataset asset a file representation, why do we need a separate asset type for files?

Reply:
Because an `file` asset cannot be rendered as a `table` :)